### PR TITLE
navigation to homepage done

### PR DESCRIPTION
**Pull request by Dulya**
Page navigation for the issue #1, I've solved this to get a basic idea on how the development goes.
Used navigator for both android and ios.

### CODE REVIEW

**Comment by berlintam** [comment]
wow, thanks so much for this @Dulya !

@starkcoffee can you take the lead on checking this over? (or maybe we can go over it on Monday)

**Comment by starkcoffee** [comment]
Cool! I can review on weekend and summarize to you Tam?

File: index.android.js [hunk 1]
```diff
@@ -22,6 +23,8 @@ var SoundDrop = React.createClass({
 		switch (route.id) {
       case 'welcome':
         return (<Main navigator={navigator}/>);
+	  case 'home':
```
Comment (starkcoffee) [inline index.android.js]: Just need to fix the indentation here.

**Comment by starkcoffee** [comment]
Thanks for figuring out how to do navigation! I wonder about the additional constructor() and navigate() functions in the Main component. I notice that the Main component uses `onPress={()=>this.navigate('home')}` and the Home component uses `onPress={()=>this.props.navigator.pop()}`. One of them uses a convenience function and one of them doesn't. It would be good to be consistent.

**Comment by Dulya** [comment]
I've fixed the indentation issue. It took me a while to figure out the consistency issue. Now I've fixed that by removing constructor and navigate functions, so that a single push line now handles the navigation in the render function.
I'm still learning react native, and thank you so much for the help and pointing out the issues in the code.

**Comment by starkcoffee** [comment]
Woops, I need to check my github notifications, I didn't see this was updated. Thanks a lot!

**Pull request merged**